torch/csrc/jit/passes/quantization.cpp

```diff
@@ -748,23 +730,38 @@ void checkGetQParamsResult(const IValue& qparams) {
   }
 }
 
-std::tuple<IValue, IValue> InsertQuantDeQuantHelper::getQParams(
+QParamMap InsertQuantDeQuantHelper::getQParamMap(
     script::Module& module, Value* v) {
   TORCH_INTERNAL_ASSERT(v->type()->isSubtypeOf(TensorType::get()));
   auto observer_name = findObserverName(v);
   TORCH_INTERNAL_ASSERT(
       observer_name,
-      "getQParams expects the corresponding observer for ",
+      "getQParamMap expects the corresponding observer for ",
       v->debugName(),
       " exists.");
   auto observer_module = module.attr(observer_name.value()).toModule();
   auto get_qparams = observer_module.get_method("get_qparams");
-  IValue qparams = get_qparams(std::vector<IValue>());
-  checkGetQParamsResult(qparams);
+  IValue result = get_qparams(std::vector<IValue>());
+  checkGetQParamsResult(result);
   auto scalar_type = observer_module.attr("dtype");
   TORCH_CHECK(scalar_type.toScalarType() != at::ScalarType::Undefined,
               "dtype of observer can't be undefined");
-  return std::make_tuple(qparams, scalar_type);
+  auto tp = result.toTuple();
+  at::Tensor scale = tp->elements()[0].toTensor().to(at::kFloat);
+  at::Tensor zero_point = tp->elements()[1].toTensor().to(at::kInt);
+  std::unordered_map<std::string, IValue> qparams = {
+    {"_scalar_type", scalar_type},
+  };
+  // TODO: get this from observer_module
```

Comment:
What does this comment mean? Aren't we getting it from observer module when we're calling its `get_qparams` method?

Reply:
oh, I mean we get qscheme from `observder_module.attr("qscheme")' and from qscheme we'll know if it is per channel, I'll clarify in the comments.